test/int/nnc/mpsdnn.tests.c

```diff
@@ -2276,6 +2276,234 @@ TEST_CASE("broadcasting semantics for mul backward (no input grad) for a")
 	ccv_nnc_tensor_free(gdb);
 }
 
+TEST_CASE("downsample bilinear NCHW")
```

Comment:
Looks like we have these in `upsample.tests.c`. Can you move these TEST_CASE there?